.gitlab/common/macos.yml

```
@@ -6,6 +6,13 @@
   - DD_API_KEY="$("$CI_PROJECT_DIR"/tools/ci/fetch_secret.sh "$AGENT_API_KEY_ORG2" token)" || exit $?; export DD_API_KEY
   - dda inv -- -e macos.report-versions -l python || true
   - dda inv -- -e macos.report-versions -l go || true
+  # Remove inactive versions
+  - |
+    if [ "$((RANDOM%20))" -eq 0 ]; then
+      echo Trying to remove inactive versions
+      dda inv -- -e macos.remove-inactive-versions -l python -t "$PYTHON_VERSION" --no-dry-run
```

```suggestion
      dda inv -- -e macos.remove-inactive-versions -l python -t "$PYTHON_VERSION"
```
I suggest to reverse the option (so set `--dry-run` when you need it only)